boilerplate/package.json

```diff
@@ -81,6 +81,9 @@
     "eslint": "8.17.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-config-standard": "17.0.0",
+    "eslint-plugin-import": "2.26.0",
```

Comment:
That's so odd, do we know why we could have been depending on these but they weren't installed by the package.json?

Reply:
It's possible I removed some too aggressively at some point

Reply:
Yeah, i couldn't run `yarn lint` in an ignited app from my branch so i had to add those back in yesterday when i was working on this

Reply:
@frankcalise part of what I'm surprised about is that tests didn't catch this. should we run commands like lint in an ignited app on CI?

Reply:
hmm well it is inside the `ignite-new.test.ts` so maybe that isn't failing like we'd expect?


```
      // #region Assert package.json Scripts Can Be Run
      // run the tests; if they fail, run will raise and this test will fail
      await run(`bun run test`, runOpts)
      await run(`bun run lint`, runOpts)
      await run(`bun run compile`, runOpts)
      expect(await run("git diff HEAD --no-ext-diff", runOpts)).toContain("+  Bowser: undefined")
      // #endregion
```

Reply:
It looks like these are required as part of `eslint-config-standard`, which I guess is what `standard` in the eslintConfig refers to?

so I'm not sure why that wouldn't come up for that test, at a glance.

Reply:
@frankcalise FYI I figured this out; CI works because it's using bun as the package manager.

> Running bun install will: Install all dependencies, devDependencies, and optionalDependencies. Bun will install peerDependencies by default.

https://bun.sh/docs/cli/install

If yarn's more likely to fail at steps like that, should we switch the test to use yarn instead of bun? or test both?

Reply:
I also don't love this behavior for bun, people don't pay close attention to `peerDependencies` because it's often useless and especially in RN libs, I see out-of-date versions referenced.

Reply:
I'm not in love with bun either, I think it got switched along the way to speed up the tests. But if they're not working correctly with it than I'm open to switching it back as it only makes sense.

Bun version on the CI is probably pinned from a while ago, so it might also be worth trying with a more updated version (but if I recall, we were following the version found on [EAS Servers](https://docs.expo.dev/build-reference/infrastructure/))